frontend/src/charts/LineChart.svelte

```diff
@@ -108,9 +107,12 @@
       transform={`translate(0,${innerHeight})`}
     />
     <g class="y axis" use:axis={yAxis} />
-    <g filter={hasFutureData ? "url(#futureColorFilter)" : ""}>
+    <g>
```

Comment:
Since you've moved the `filter` property into the children elements, you can remove this `<g />` entirely. I added it in the previous commit but it's not needed anymore.

Reply:
Thanks :)